lib/live-collection.js

```diff
@@ -255,6 +254,18 @@ module.exports = Backbone.Collection.extend({
     }, PATCH_TIMEOUT);
   },
 
+  resubscribeOnModelChange: function (model, prop){
+    var evtName = prop ? 'change:' + prop : 'change';
+    this.listenTo(model, evtName, this.resubscribe, this);
+  },
+
+  resubscribe: function (){
+    this.reset();
+    this.subscription.cancel();
+    this.subscription = null;
+    this.listen({ snapshot: true });
```

Comment:
The only thing I'm worried about here is that the original listen options will not be preserved. So, for example, if you chose not to use snapshots in the original subscribe, when the model change happens, a snapshot will be pulled...